src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -122,6 +183,116 @@ def _load_from_data(self, data):
                 if self._should_include_example(d):
                     self.examples.append(HelpExample(d))
 
+    # get data object from parsed yaml
+    def _load_from_parsed_yaml(self, data):
+        content = data.get("content")
+        info_type = None
+        info = None
+        new_data = {}
+
+        for elem in content:
+            for key, value in elem.items():
+                # find the command / group's help text
+                if value.get("name") and value.get("name") == self.command:
+                    info_type = key
+                    info = value
+                    break
```

Comment:
Two lookups down to one?

```python
try:
    if value["name"] == self.command:
        info_type = key
        info = value
        break
except KeyError:
    pass
```
* https://blogs.msdn.microsoft.com/pythonengineering/2016/06/29/idiomatic-python-eafp-versus-lbyl/